docs/deployment/feature-gates.md

```diff
@@ -22,7 +22,7 @@ The following tables are a summary of the feature gates that you can set on etcd
 
 | Feature          | Default | Stage   | Since  | Until |
 |------------------|---------|---------|--------|-------|
-| `UseEtcdWrapper` | `false` | `Alpha` | `0.19` |       |
+| `UseEtcdWrapper` | `true`  | `Beta`  | `0.22` |       |
```

Comment:
```suggestion
| `UseEtcdWrapper` | `false` | `Alpha` | `0.19` | `0.21` |
| `UseEtcdWrapper` | `true`  | `Beta`  | `0.22` |        |
```

All stages of a feature gate need to be captured in this doc, similar to https://github.com/gardener/gardener/blob/master/docs/deployment/feature_gates.md